### PR TITLE
Pull through translatability

### PR DESCRIPTION
**Pull request by annaxuphoto**
# Description
Adding raw strings in the Code Studio pull through to our translation pipeline

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: curricula/templates/curricula/partials/code_studio_pull_through/level_contents.html [hunk 1]
```diff
@@ -38,16 +38,16 @@ <h3>Student Instructions</h3>
...
     {% endif %}
 
     <!-- Curriculum Reference level -->
     {% if level.type == "LevelGroup" %}
     <div class="level-group">
         <p>
-            This level is an assessment or survey with multiple questions.
-            To view this level click the "View on Code Studio" link.
+            {% trans "This level is an assessment or survey with multiple questions." %}
+            {% blocktrans %}To view this level click the "View on Code Studio" link.{% endblocktrans %}
```
Comment (bethanyaconnor) [inline curricula/templates/curricula/partials/code_studio_pull_through/level_contents.html:50]: Why does this one need the `blocktrans` tag?

File: curricula/templates/curricula/partials/code_studio_pull_through/level_numbered_tab_sublevel.html [hunk 2]
```diff
@@ -1,3 +1,4 @@
+{% load i18n %}
```
Comment (bethanyaconnor) [inline curricula/templates/curricula/partials/code_studio_pull_through/level_numbered_tab_sublevel.html:1]: Is this needed?

**Review by bethanyaconnor — APPROVED** [review]

File: curricula/templates/curricula/partials/code_studio_pull_through/level_contents.html (hunk 1 above)
Comment (annaxuphoto) [reply curricula/templates/curricula/partials/code_studio_pull_through/level_contents.html:50]: `blocktrans` auto-escapes characters (in this case the double quotes)

File: curricula/templates/curricula/partials/code_studio_pull_through/level_numbered_tab_sublevel.html (hunk 2 above)
Comment (annaxuphoto) [reply curricula/templates/curricula/partials/code_studio_pull_through/level_numbered_tab_sublevel.html]: nope, removed!

**Pull request merged**